ckanext/data_qld/helpers.py

```diff
@@ -135,12 +135,6 @@ def format_activity_data(data):
 
 # Data.Qld specific comments helper functions
 
-def get_datarequest_comments_badge(datarequest_id):
```

Comment:
is this function being deleted or moved to another plugin?

Reply:
Hi @duttonw - yes, in order to decouple from `ckanext-data-qld` it's been moved:

- [DQL2-24] Refactored get_datarequest_comments_badge to get_content_type_comments_badge so it can be used for different content_types
- [DQL2-24] Removed comment badge helpers and moved them into ytp-comments